src/test/java/org/openbroker/cloudevents/CloudEventsJavaTest.java

```diff
@@ -44,7 +44,7 @@ public void testCloudEventSerializationComplexType() throws JsonProcessingExcept
 
         String json = new ObjectMapper().writeValueAsString(event);
         CloudEvent<ComplexClass> deserialized = SerializationKt.cloudEvent(json, ComplexClass.class);
-        assertTrue(deserialized.getData() instanceof ComplexClass);
+        assertNotNull(deserialized.getData());
```

Comment:
You change the meaning of the test.

Reply:
I understand what you wanted to check. But, what you did in this test was to check the correctness of the work of the Java machine and not of your code :-). Because If you use Generic and the deserialization goes to a specific class, then in any case you get one of three options:

1. DeserializingException - error during deserialization
2. Return null
3. Return to CloudEvent <ComplexClass>

In other words, with such an operation, the function cannot return anything like CloudEvent <Integer> or CloudEvent <String>

Reply:
That's true, I think this is a good alternative. All tough I would have preferred this changes to test cases to be in a pull request of it's own, rather than together with changes in the build script.